### ISSUE TITLE
Breaks on inline CSS with brace selector

### ISSUE THREAD

**Issue by bavarianbytes** [jenstornell/kirby-html-minifier#17]
Hi,

my site echos inline css in the page head. This CSS uses braces on some selectors, like `a > span`.
The plugin seams to cuts away all of the CSS beginning with the first `>` in finds.

Greetz

**Comment by jenstornell** [comment]
Yes, I the `<` and `>` is a problem if it's inside inline style or inline js. As a workaround for now, maybe you can move the css into your css file?

**Comment by bavarianbytes** [comment]
Ok, good to know.
As a workaround i disabled the plugin :-). Cause i need inline CSS for performance reasons.

**Comment by jenstornell** [comment]
That workaround should work as well. 😄 

Related: https://github.com/jenstornell/kirby-html-minifier/issues/13
